### PR TITLE
✨ Add log in and sign in links +story

### PR DESCRIPTION
**Pull request by aronvaupel**
Applied most suggestions. Thanks
Pls check, if the whole route/switch/link construct is used correctly, also in context of further use of component. 

 issue #7

### CODE REVIEW

File: src/app/components/LogIn-SignIn-Link/LogIn-SignIn-Link.module.css [hunk 1]
```diff
@@ -0,0 +1,13 @@
+.LogInSignInLinkWrapper {
```
Comment (phlgr) [inline src/app/components/LogIn-SignIn-Link/LogIn-SignIn-Link.module.css:1]:
```suggestion
.wrapper {
```

File: src/app/components/LogIn-SignIn-Link/LogIn-SignIn-Link.tsx [hunk 2]
```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import styles from './LogIn-SignIn-Link.module.css';
+
+export type LogInSignInLinkProps = {
+  href: string;
+  value: 'Log in' | 'Sign in';
+};
+
+const LogInSignInLink = ({
```
Comment (phlgr) [inline src/app/components/LogIn-SignIn-Link/LogIn-SignIn-Link.tsx:9]: This component seems to be pretty specific, why not a generic Link component that can receive any text and display it?

File: src/app/components/LogIn-SignIn-Link/LogIn-SignIn-Link.tsx [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+  href,
+  value,
+}: LogInSignInLinkProps): JSX.Element => {
+  return (
+    <div className={styles.LogInSignInLinkWrapper}>
+      <a className={styles.LogInSignInLink} href={href}>
+        {value}
+      </a>
```
Comment (phlgr) [inline src/app/components/LogIn-SignIn-Link/LogIn-SignIn-Link.tsx:17]: Don't use the `a`-Tag here. Please take a look here: https://reactrouter.com/web/api/Link

**Review by phlgr — CHANGES_REQUESTED** [review]

File: src/app/components/LogIn-SignIn-Link/LogIn-SignIn-Link.tsx (hunk 2 above)
Comment (aronvaupel) [reply src/app/components/LogIn-SignIn-Link/LogIn-SignIn-Link.tsx:9]: Appears only once, other links are in LinkIcons, specific naming to make them distinguishable.

File: src/app/components/LogIn-SignIn-Link/LogIn-SignIn-Link.tsx [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+const LogInSignInLink = ({ value, url }: LogInSignInLinkProps): JSX.Element => {
+  return (
+    <div className={styles.wrapper}>
+      <Route>
+        <Switch>
+          <Link to={url} className={styles.Link}>
+            {value}
+          </Link>
+        </Switch>
+      </Route>
```
Comment (phlgr) [inline src/app/components/LogIn-SignIn-Link/LogIn-SignIn-Link.tsx:19]: The Route and Switch should be in the App.tsx.

**Review by phlgr — CHANGES_REQUESTED** [review]

**Pull request merged**